package.json

```diff
@@ -85,5 +87,10 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "husky": {
+    "hooks": {
+      "post-merge": "npm run version:minor"
```

Comment:
Does this run on the CI? This will only run in your machine when manually merging, am I right?